tests/yast2_gui/yast2_users.pm

```diff
@@ -16,11 +16,107 @@ use strict;
 use warnings;
 use testapi;
 
+use utils qw(type_string_slow_extended);
+use version_utils qw(is_tumbleweed is_leap);
+
+sub write_change_now {
+    send_key 'alt-x';
+    send_key 'down';
+    send_key 'down';
+    send_key 'down';
+    send_key 'ret';
+}
+
+sub check_with_xterm {
+    my $command = shift;
+    x11_start_program('xterm');
+    become_root;
+    assert_script_run($command);
```

Comment:
another neat pick, you may use [validate_script_output](https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L1102) or even better because you know the result you can use cpan's *Test::Assert 'assert_equals';*. 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/validation/ibft.pm#L99
Nice stuff, you can even have an additional note to the comparison.